bus/bus.go

```diff
@@ -1808,15 +1851,15 @@ func (b *bus) handlePOSTAlertsDismiss(jc jape.Context) {
 	if jc.Decode(&ids) != nil {
 		return
 	}
-	jc.Check("failed to dismiss alerts", b.alertMgr.DismissAlerts(jc.Request.Context(), ids...))
+	jc.Check("failed to dismiss alerts", b.alerts.DismissAlerts(jc.Request.Context(), ids...))
 }
 
 func (b *bus) handlePOSTAlertsRegister(jc jape.Context) {
 	var alert alerts.Alert
 	if jc.Decode(&alert) != nil {
 		return
 	}
-	jc.Check("failed to register alert", b.alertMgr.RegisterAlert(jc.Request.Context(), alert))
```

Comment:
This might be the reason for the origin change? because you switched to `alertMgr` from `alerts`.